### DISCUSSION TITLE
Setting up project on Windows 10

### DISCUSSION THREAD

**Discussion by sisonsco** [bolt/core#2456] [Q&A]
Hello,

I am currently trying to start a new Bolt 4.0 project on Windows 10 but am having trouble figuring out what to do. I am on step four on the installation documentation, where i need to run `start bin/console bolt:setup`. However, The console file does not give a file extension. What should I run it with?

For reference if needed, I am using XAMPP v3.2.4. 
Thanks

**Comment by mcbmcb0** [comment]
i'm stuck on this too. did you find a solution?

EDIT:
using xampp control panel (as administrator) run the shell.
type this: `php full/path/to/bolt/install/bin/console bolt:setup`
then follow the setup steps.

note I did set up an alias in xampp's httpd.conf first as for me apache is serving outside of the xampp file structure:
`alias bolt_test full/path/to/bolt/install`
tho not sure this was necessary
you don't have to create the db first (in fact, don't).

now, on the the next error (Failed to write cache file...)
EDIT2 - reloaded (again) and cache error no longer - a working install!
